.github/workflows/wheels.yml

```diff
@@ -0,0 +1,43 @@
+name: Build-wheels
+on:
+  release:
+  workflow_dispatch:
+  
+jobs:
+  build_wheels:
+    name: Build wheels for cp${{ matrix.python }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python: [37, 38, 39]
+        include:
+          - os: ubuntu-latest
+            cibw_before_all: yum install -y openblas-devel
+            cibw_environment: CBLAS_ROOT=/usr/include/openblas
+          - os: macos-latest
+            cibw_environment: MACOSX_DEPLOYMENT_TARGET=10.14 # Required for C++17 support
```

Comment:
This will fail when `macos-latest` points to Big Sur, right?  If so, should we specify `macos-10.15` explicitly?

Reply:
No, it shouldn't. The documentation [here](https://cibuildwheel.readthedocs.io/en/stable/cpp_standards/) and [here](https://cmake.org/cmake/help/latest/envvar/MACOSX_DEPLOYMENT_TARGET.html) says that this refers to the minimum supported macOS version

Reply:
@brownj85 out of curiosity, is it relatively easy to support macos arm as well? I'm just thinking about the new M1 chip

Reply:
@josh146, in general yes, though we'd need a self-hosted runner to do automated testing. However the wheels wouldn't be pip installable because numba (via thewalrus) does not support macos arm (see numba/llvmlite#693).